apps/backend/src/poll/poll/poll.service.ts

```diff
@@ -33,6 +36,29 @@ export class PollService {
     ) {
     }
 
+  async onModuleInit() {
+    const participants = await this.participantModel.find({
+      $or: [
+        {participation: {$exists: true}},
+        {indeterminateParticipation: {$exists: true}},
+      ]
+    }).exec();
+    for (const participant of participants) {
+      participant.selection ||= {};
+      for (const participation of (participant as any).participation || []) {
+        participant.selection[participation.toString()] = 'yes';
+      }
+      for (const participation of (participant as any).indeterminateParticipation || []) {
+        participant.selection[participation.toString()] = 'maybe';
+      }
+      participant.markModified('selection');
+      participant.set('participation', undefined);
+      participant.set('indeterminateParticipation', undefined);
+    }
+    await this.participantModel.bulkSave(participants, {timestamps: false});
+    this.logger.log(`Migrated ${participants.length} participants to the new selection format.`);
```

Comment:
This should only be logged, when a migration occured.
Currently it would also log something like:
> Migrated 0 participants to the new selection format

But since this message has no added value you can abandon it.